### PR TITLE
fix: `fullAppDisplay` album info & background

### PR DESCRIPTION
**Pull request by huhridge**
As the hermes protocol is broken `1.1.81` onwards, this can "fix" the issue. Now the lyrics-plus background will always match spotify lyrics background.

Maybe, in the future, there is a way to restore the old colorextractor.

### CODE REVIEW

**Comment by huhridge** [comment]
NOTE: It only works for songs which also have lyrics on spotify, but imo it is better than nothing, i will find a better solution when i have some time

File: CustomApps/lyrics-plus/index.js [hunk 1]
```diff
@@ -162,8 +162,13 @@ class LyricsContainer extends react.Component {
     async fetchColors(uri) {
         let prominent = 0;
         try {
-            const colors = await CosmosAsync.get(`hm://colorextractor/v1/extract-presets?uri=${uri}&format=json`);
-            prominent = colors.entries[0].color_swatches[4].color;
+            // const colors = await CosmosAsync.get(`hm://colorextractor/v1/extract-presets?uri=${uri}&format=json`);
+            // prominent = colors.entries[0].color_swatches[4].color;
+            // hermes deprecated 1.1.81 onwards
```
Comment (afonsojramos) [inline CustomApps/lyrics-plus/index.js:167]: Why did you choose to leave this commented and not delete it?

File: Extensions/fullAppDisplay.js [hunk 2]
```diff
@@ -341,9 +341,13 @@ body.video-full-screen.video-full-screen--hide-ui {
 
         async getAlbumDate(uri) {
             const id = uri.replace("spotify:album:", "");
-            const albumInfo = await Spicetify.CosmosAsync.get(`hm://album/v1/album-app/album/${id}/desktop`);
+            // const albumInfo = await Spicetify.CosmosAsync.get(`hm://album/v1/album-app/album/${id}/desktop`);
+            // const albumDate = new Date(albumInfo.year, (albumInfo.month || 1) - 1, albumInfo.day || 0);
+            // hermes protocol deprecated 1.1.81 onwards
```
Comment (afonsojramos) [inline Extensions/fullAppDisplay.js:346]: Same here.

**Review by afonsojramos — APPROVED** [review]
Overall I agree with the implementations that you propose. I would even say that it is not a temporary solution, since it will probably work more reliably than the previous one!

If we need the previous code, we'll check the git history, no need to leave old code commented 😉

File: CustomApps/lyrics-plus/index.js (hunk 1 above)
Comment (huhridge) [reply CustomApps/lyrics-plus/index.js:167]: That was just me being lazy, so I wouldn't forget how it worked before, should I remove it?

Comment (afonsojramos) [reply CustomApps/lyrics-plus/index.js:167]: Yeah, as I said on the review: "If we need the previous code, we'll check the git history, no need to leave old code commented 😉"

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**